design/user_types.go

```diff
@@ -129,3 +129,17 @@ var UpdateTrackerQueryAlternatePayload = a.Type("UpdateTrackerQueryAlternatePayl
 	})
 	a.Required("query", "schedule", "trackerID")
 })
+
+// IdentityDataAttributes represents an identified user object attributes
+var IdentityDataAttributes = a.Type("IdentityDataAttributes", func() {
+	a.Attribute("fullName", d.String, "The users full name")
+	a.Attribute("imageURL", d.String, "The avatar image for the user")
+})
+
+// IdentityData represents an identified user object
+var IdentityData = a.Type("IdentityData", func() {
```

Comment:
@tsmaeder , as we are following `jsonapi spec`, it is more readable that `*Data` defines the top level data attribute for the given API.
I would suggest doing `*DataForUpdate` or `*DataForCreate` etc, because I faced a scenario where `Create payload` looks different than `Update payload` (required attributes vary, for e.g: a relationship must be given while creation but not necessarily while update).
So I defined `WorkItemDataForUpdate` [here](https://github.com/almighty/almighty-core/pull/482/files#diff-aa2cec7dc60fd15e23552b59d51fdbdcR49).